packages/account/src/Components/poi/poi-confirm-with-example-form-container/__tests__/poi-confirm-with-example-form-container.spec.tsx

```diff
@@ -59,8 +57,7 @@ describe('<PoiConfirmWithExampleFormContainer/>', () => {
     it('should render PersonalDetailsForm with image and checkbox', async () => {
         render(<PoiConfirmWithExampleFormContainer {...mock_props} />);
 
-        expect(await screen.findByText('PoiNameDobExampleImage')).toBeInTheDocument();
```

Comment:
Why can't we maintain the tests instead of removing them?

Reply:
this was mocked imported svg. and now we don't import it.
let me try to find the image and try to check it in the test 🙏